src/helpers/schema.cjs

```diff
@@ -78,11 +78,11 @@ const reportSchema = {
 			unevaluatedProperties: false,
 			properties: {
 				framework: { $ref: '/testReporting/nonEmptyUnpaddedString' },
-				lmsBuild: {
+				lmsBuildNumber: {
 					type: 'string',
-					pattern: '([0-9]{2}\\.){3}[0-9]{5}'
+					pattern: '([0-9]{2}\\.){2}[0-9]{1,2}\\.[0-9]{5}'
```

Comment:
The original pattern didn't match build like `20.24.1.12345` it would only match `20.24.12.12345`. It was assuming the first 3 build parts were always 2 and the 3 part can be 1 or 2 in length.